docs/source/en/model_doc/gpt_neox_japanese.mdx

```diff
@@ -0,0 +1,66 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# GPT-NeoX-Japanese
+
+## Overview
+
+We introduce GPT-NeoX-Japanese, which Japanese autoregressive language model trained by using [https://github.com/EleutherAI/gpt-neox](https://github.com/EleutherAI/gpt-neox).
+Japanese is a special language consisting of a large vocabulary and a combination of hiragana, katakana, and kanji.
+To address the distinctive structure of the Japanese language, we use a [special sub-word tokenizer](https://github.com/tanreinama/Japanese-BPEEncoder_V2). (Many thanks to open this kind of great tokenizer)
+Furthermore, we removed bias parameters from the transformer blocks following [PaLM from Google](https://ai.googleblog.com/2022/04/pathways-language-model-palm-scaling-to.html).
+
+Development of the model was led by Shinya Otani, Takayoshi Makabe, Anuj Arora, and Kyo Hattori from [ABEJA, Inc.](https://www.abejainc.com/)
```

Comment:
I think we can link our github profiles here [Shinya Otani] (https://github.com/SO0529)